certified-connectors/BiG EVAL/apiDefinition.swagger.json

```diff
@@ -0,0 +1,513 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "BiG EVAL",
+    "description": "BiG EVAL Data Quality Automation platform for data validation and data test automation.",
+    "version": "1.0",
+    "contact": {
+      "name": "BiG EVAL Support",
+      "url": "https://support.bigeval.com/",
+      "email": "support@bigeval.com"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://bigeval.com/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://bigeval.com/company/privacy/"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Data;Business Intelligence"
+    }
+  ],
+  "host": "bigeval-lttbo01.eu.ngrok.io",

```

Comment:
@boltth can you please confirm that this is a production environment host URL?

Reply:
BiG EVAL is an on-premise solution and every customer will configure his own public URL. Therefore no public available production-URL exists. We will change this to anything like "bigeval.mycompany.com"